packages/now-cli/src/util/input/input-root-directory.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import inquirer from 'inquirer';
+
+export async function inputRootDirectory(autoConfirm: boolean) {
+  if (autoConfirm) {
+    return null;
+  }
+
+  const { rootDirectory } = await inquirer.prompt({
+    type: 'input',
+    name: 'rootDirectory',
+    message: `In which directory is your code located?`,
+    transformer: (input: string) => {
+      return input ? input : '[.] ';
+    },
+  });
+
+  if (!rootDirectory || rootDirectory === '.' || rootDirectory === './') {
```

Comment:
This is missing cases where `rootDirectory` is `././`, `././.`, etc... We should run these comparisons on the normalized `rootDirectory`.